app/controllers/spree/admin/orders_controller_decorator.rb

```diff
@@ -11,9 +11,7 @@ def update_order_on_netsuite
       end
 
       def need_to_update_on_netsuite
-        ActiveRecord::Base.connected_to(role: :writing) do
```

Comment:
need to revert this code and add check it project has two database link primary and replica.